modules/exploits/windows/http/file_sharing_wizard_seh.rb

```diff
@@ -0,0 +1,57 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Seh
+
+  def initialize(info = {})
+    super update_info(info,
+                      'Name' => 'File Sharing Wizard 1.5.0 - POST SEH Overflow',
+                      'Description' => %q(
+        This module exploits a SEH based buffer overflow in an HTTP POST parameter in File Sharing Wizard 1.5.0
+      ),
+                      'Author' => [
+                        'x00pwn', # Original exploit
+                        'Dean Welch <dean_welch[at]rapid7.com>' # Module
+                      ],
+                      'License'        => MSF_LICENSE,
+                      'References'     =>
+                          [
+                            %w(CVE 2019-16724),
+                            %w(EDB 47412)
+                          ],
+                      'Payload' =>
+                          {
+                            BadChars: "\x00\x20"
```

Comment:
I'm pretty sure these hash keys should be strings.

Reply:
I had it as a string since that's what most other modules seem to do but rubocop was complaining saying this was the preferred way

Reply:
I'm pretty sure the API expects a string. I don't think there's any normalization from symbol to string. I'd check to make sure your code still works this way.

Reply:
```
msf5 exploit(windows/http/file_sharing_wizard_seh) > pry
[*] Starting Pry shell...
[*] You are in exploit/windows/http/file_sharing_wizard_seh

[1] pry(#<Msf::Modules::Exploit__Windows__Http__File_sharing_wizard_seh::MetasploitModule>)> payload_badchars
=> nil
[2] pry(#<Msf::Modules::Exploit__Windows__Http__File_sharing_wizard_seh::MetasploitModule>)>
msf5 exploit(windows/http/file_sharing_wizard_seh) > edit
[*] Launching vim -i NONE /rapid7/metasploit-framework/modules/exploits/windows/http/file_sharing_wizard_seh.rb
msf5 exploit(windows/http/file_sharing_wizard_seh) > reload
[*] Reloading module...
msf5 exploit(windows/http/file_sharing_wizard_seh) > pry
[*] Starting Pry shell...
[*] You are in exploit/windows/http/file_sharing_wizard_seh

[1] pry(#<Msf::Modules::Exploit__Windows__Http__File_sharing_wizard_seh::MetasploitModule>)> payload_badchars
=> "\x00 "
[2] pry(#<Msf::Modules::Exploit__Windows__Http__File_sharing_wizard_seh::MetasploitModule>)>
```

Reply:
Confirmed that there is no normalization. Badchars aren't set presently.